holoviews/plotting/renderer.py

```diff
@@ -298,7 +299,9 @@ def get_widget(self_or_cls, plot, widget_type, **kwargs):
         if not isinstance(plot, Plot):
             plot = self_or_cls.get_plot(plot)
         dynamic = plot.dynamic
-        if widget_type == 'auto':
+        if widget_type == 'server':
```

Comment:
I would rather avoid introducing a new widget type - why not just switch to bokeh widgets if the renderer is in 'server' mode?